image/Make.package

```diff
@@ -94,6 +94,7 @@ msdos:
 	do \
 		[ -d $(DESTDIR)/$$f ] && mmd -i $(MSDOS_IMAGE) ::$$f; \
 		[ -f $(DESTDIR)/$$f ] && mcopy -i $(MSDOS_IMAGE) $(MSDOS_COPYOPTS) $(DESTDIR)$$f ::$$f; \
+		true; \
```

Comment:
This is error masking. The Makefile **has** to fail whenever the `mcopy` command returns an unexpected exit code. Please investigate the reason why `mdir` returns a non-zero code and add a check for that code.